cmd/supportarchive/cmd.go

```diff
@@ -67,10 +68,12 @@ func New() *cobra.Command {
 				return nil
 			}
 
-			return fmt.Errorf("%s %s %s\n",
-				"The only option to retrieve the support archive is by using '--stdout=true'.",
-				"Please provide this parameter and make sure that you pipe the command output to a file.",
-				"Otherwise, your terminal will be flooded with binary data.")
+			sb := strings.Builder{}
+			sb.WriteString("The only option to retrieve the support archive is by using '--stdout=true'. ")
```

Comment:
why do u need extra space hee?
```suggestion
			sb.WriteString("The only option to retrieve the support archive is by using '--stdout=true'.")
```

Reply:
Because in line 76 (where `sb.String()` is invoked), three separate sentences are connected. Removing the spaces would make the output look like this, making it harder to recognize when new sentences (or words) begin/end:
```Error: The only option to retrieve the support archive is by using '--stdout=true'.Please provide this parameter and make sure that you pipe the command output to a file.Otherwise, your terminal will be flooded with binary data.```